fcl/dunefd/reco/standard_reco_dune10kt_1x2x6_light.fcl

```diff
@@ -0,0 +1,15 @@
+#include "standard_reco2_dune10kt_1x2x6.fcl"
+
+physics:
+{
+  @table::physics
+
+   #  Set up of PDS only reconstruction in 1x2x6 geo
+  reco: 
+  [
+    @sequence::dunefd_horizdrift_pd_reco1,
+    @sequence::dunefd_horizdrift_pd_reco2,
+    rns
+  ]
+
+}
```

Comment:
This could collapse to
physics.reco: [ @sequence::dunefd_horizdrift_pd_reco1, @sequence::dunefd_horizdrift_pd_reco12, rns ]

Reply:
Added a commit with your suggested changes